Sources/Datadog/Core/Utils/DDError.swift

```diff
@@ -29,3 +30,15 @@ internal struct DDError {
         }
     }
 }
+
+private func isNSErrorOrItsSubclass(_ error: Error) -> Bool {
```

Comment:
i guess á-la-Cocoa naming of this method would be `isKindOfNSError`

Reply:
Could be 🙂, but it's a minor nit. It's more verbose now.